test/FluentMigrator.Tests/Integration/MigrationRunnerTests.cs

```diff
@@ -1962,10 +1946,15 @@ internal class TestIndexNamingConventionWithSchema : Migration
     {
         public override void Up()
         {
-            var createSchemaExpr = Create.Schema("TestSchema");
+            // SQLite doesn't support creating schemas so for non SQLite DB's we'll create
+            // the schema, but for SQLite we'll attach a temp DB with the schema alias
+            var createSchemaExpr = IfDatabase(t => t != "SQLite").Create.Schema("TestSchema");
```

Comment:
I was surprised to see we dont have a class with all the processor id's define. e.g.,
```c#
public static class ProcessorId
{
  SQLite = nameof(SQLite);
}
```
and then this code would read: 
```c#
var createSchemaExpr = IfDatabase(t => t != ProcessorId.SQLite).Create.Schema("TestSchema");
```
and SqliteProcessor would be updated so that line 50 is https://github.com/fluentmigrator/fluentmigrator/blob/9c87d698b161b3a1bb596a5624c02891a64459c5/src/FluentMigrator.Runner.SQLite/Processors/SQLite/SQLiteProcessor.cs#L50 :
```c#
public override string DatabaseType => ProcessorId.SQLite;
```

Reply:
Where would you see the `ProcessorId` class living as if it's in core it's not then extendable by custom provider implementations. Unless we are only really interested here in Core supported providers (which I think is reasonable)

Reply:
The `FluentMigrator` project made most sense to me so I've put that in place and updated the majority of places